examples/toolchains/python/README.md

```diff
@@ -0,0 +1,13 @@
+Python Toolchain Example
+========================
+
+This is an example Python project with modules that uses `rules_python`. Providing Python modules without Nix is currently not supported.
```

Comment:
> Providing Python modules without Nix is currently not supported.

That phrasing seems a bit misleading. This example currently doesn't illustrate the non-Nix setup. However, rules_python is capable of installing Python modules [through pip](https://github.com/bazelbuild/rules_python#installing-pip-dependencies). It's just not as hermetic as the Nix provided ones, because the pip installation runs in a repository rule and is therefore not sandboxed, does not have access to the Python toolchain provided through Bazel's platforms and toolchains, and especially for Python modules with native dependencies could link against arbitrary system installed libraries.